docs/reference/settings.md

```diff
@@ -145,8 +145,8 @@ Note that all languages that exist in ``WAGTAIL_CONTENT_LANGUAGES`` must also ex
 
 ## Embeds
 
-Wagtail supports generating embed code from URLs to content on an external providers such as Youtube or Twitter. By default, Wagtail will fetch the embed code directly from the relevant provider's site using the oEmbed protocol.
-Wagtail has a builtin list of the most common providers.
+Wagtail supports generating embed code from URLs to content on external providers such as YouTube or X (Formally Twitter). By default, Wagtail will fetch the embed code directly from the relevant provider's site using the oEmbed protocol.
```

Comment:
```suggestion
Wagtail supports generating embed code from URLs to content on external providers such as YouTube or X (formerly Twitter). By default, Wagtail will fetch the embed code directly from the relevant provider's site using the oEmbed protocol.
```

Lower case as it's not a proper noun. 

Should be formerly (past tense), not formally (as in to be more official).